services/catalog/service/src/messaging/models/handler-registrator.ts

```diff
@@ -1,13 +1,13 @@
-import { LoginPgPool } from '@axinom/mosaic-db-common';
 import { RascalConfigBuilder } from '@axinom/mosaic-message-bus';
+import { RabbitMqInboxWriter } from '@axinom/mosaic-transactional-inbox-outbox';
 import { Config } from '../../common';
 
 export interface ContentTypeRegistrant {
```

Comment:
If not now, please add a task somewhere to align the naming.
`ContentTypeRegistrant` in `handler-registrator.ts` file looks very confusing to
me.

Reply:
Renamed it to be in line with the naming of where it is used.